lib/quby/questionnaires/dsl/questionnaire_builder.rb

```diff
@@ -54,6 +54,17 @@ def sbg_key(sbg_key)
           @questionnaire.sbg_key = sbg_key
         end
 
+        def sbg_domain(sbg_key, from: nil, till: nil, outcome: nil, key: nil, primary: false)
```

Comment:
`key: sbg_key` maakt de output van de json completer en is de juiste default voor alle gevallen behalve 1.